### PR TITLE
Remove Baseline from project

### PR DESCRIPTION
**Pull request by lavaldi**
Since Baseline is not a package open to the public, it was decided to work only with Tailwind

### CODE REVIEW

File: components/Nav.js [hunk 1]
```diff
@@ -1,42 +1,27 @@
...
-    <Inline as="nav" className="p-4">
-      <Inline space={5}>
-        <Icon color={Color.Neutral800}>
+    <nav className="flex flex-no-wrap flex-row items-center justify-start p-4">
+      <div className="flex flex-no-wrap flex-row items-center justify-start space-x-5">
+        <span className="text-neutral-800">
           <Menu />
-        </Icon>
+        </span>
         <Link href="/" passHref>
```
Comment (cristianbgp) [inline components/Nav.js]:
```suggestion
        <Link href="/">
```

File: components/Input.js [hunk 2]
```diff
@@ -1,119 +1,66 @@
...
-        space={2}
-        className={[
-          "border hover:border-neutral-200 focus-within:border-neutral-300",
-          "rounded-md bg-white w-full",
-          { "border-white": !enforceBorder },
-          { "border-neutral-200": enforceBorder },
-        ]}
-        p={[0, paddingX]}
+      <div
+        className={cx([
```
Comment (cristianbgp) [inline components/Input.js]: just curious if this is neccessary here?

Comment (cristianbgp) [reply components/Input.js]: Either way I think the package is a good addition

**Review by cristianbgp — APPROVED** [review]
LGTM, just some comments

File: components/Input.js (hunk 2 above)
Comment (lavaldi) [reply components/Input.js]: yeah, there was a condition there before, then I forgot to modify this 😅

**Pull request merged**